packages/wasm_run/lib/src/wasm_bindings/_wasm_interop_web.dart

```diff
@@ -821,7 +821,9 @@ GlobalTy? _getGlobalType(Object value) {
 TableTy? _getTableType(Object value) {
   final t = _getType(value);
   if (t == null) return null;
-  final ty = ValueTy.values.byName(t['element']! as String);
+  final ty = ValueTy.values.firstWhere((value) =>
+      value.toString().split('.').last.toLowerCase() ==
```

Comment:
This is great thank you. I believe we could use Enum's `.name` getter:

```suggestion
      value.name.toLowerCase() ==
```

Reply:
Updated. Do you have any thoughts about what is going on under _getType function?